### ISSUE TITLE
Bootstrap button class .btn-primary fails color contrast  

### ISSUE THREAD

**Issue by nickbristow** [18F/18f.gsa.gov#58]
The blue "RSVP for 18F Demo Day" uses the .btn-primary class from bootstrap which has, 
background - #428bca
Text - white

This fails 508 color contrast at 3.63:1 Which is just below 508 standards. 

The hover state has,
background - #3276b1
Text - White 

Which passes 508 but not AAA. 

Fix Suggestion
Override bootstrap class with new background colors
Normal state
Background - 2e6ca2

Hover
Background - 255782
